examples/stm32/xively_demo_mt/Src/httpserver-socket.c

```diff
@@ -0,0 +1,312 @@
+/**

```

Comment:
Is this file really needed?  Do we need to include the http server socket?  It has a precompiled web page in here and everything.